pkg/fs/realfs.go

```diff
@@ -43,8 +44,32 @@ func (realFS) Open(name string) (File, error) { return os.Open(name) }
 // Stat delegates to os.Stat.
 func (realFS) Stat(name string) (os.FileInfo, error) { return os.Stat(name) }
 
-// ReadFile delegates to ioutil.ReadFile.
-func (realFS) ReadFile(name string) ([]byte, error) { return ioutil.ReadFile(name) }
+var separator = []byte(`
+---
+`)
+
+// ReadFile use glob to find the matching files and then read content from all of them
+func (realFS) ReadFile(name string) ([]byte, error) {
+	output := []byte{}
+	files, err := filepath.Glob(name)
+	if err != nil || len(files) == 0 {
+		return output, nil
```

Comment:
Shouldn't we return the error if the error isn't nil?

Reply:
oops, my bad here. Will change it.

Reply:
Done